pyproject.toml

```diff
@@ -69,11 +66,10 @@ unfixable = []
 
 per-file-ignores = {}
 
-
 # Allow unused variables when underscore-prefixed.
 dummy-variable-rgx = "^(_+|(_+[a-zA-Z0-9_]*[a-zA-Z0-9]+?))$"
 
-[tool.ruff.lint.isort]
```

Comment:
I'm assuming this is because of some linter deprecation/error?

Reply:
indeed. Apparently this changed in the last half year and my precommit wouldn't even let me commit anything without the change ;-)